src/System.Private.CoreLib/src/System/Globalization/CultureInfo.CoreCLR.cs

```diff
@@ -266,5 +203,27 @@ internal static bool SetCultureInfoForUserPreferredLanguageInAppX(CultureInfo ci
             return s_WindowsRuntimeResourceManager.SetGlobalResourceContextDefaultCulture(ci);
         }
 #endif
+
+        internal static CultureInfo GetCurrentUICultureNoAppX()
```

Comment:
This should not need to be a separate method. It can direct call like in CoreRT version.

Reply:
I kept it in separate method because it was used from `ResourceManager`. I'll see if that is still necessary and how CoreRT handles it.